### PR TITLE
Update to version 1.1.3 and rdkit 2022.03, 2022.09

### PR DESCRIPTION
**Pull request by ljn917**
Update to version 1.1.3 and rdkit 2022.03, 2022.09. Also rerender.

### CODE REVIEW

File: recipe/conda_build_config.yaml [hunk 1]
```diff
@@ -1,4 +1,48 @@
...
+  - 2022.03.*
+  - 2022.03.*
+  - 2022.03.*
+  - 2022.09.*
+  - 2022.09.*
+  - 2022.09.*
+  - 2022.09.*
+zip_keys:
+  - python_version
+  - rdkit_version
```
Comment (hadim) [inline recipe/conda_build_config.yaml:48]: I don't think it's needed to enumerate the python versions and we should stick with conda-forge defaults.

Can you replace by:

```yaml
rdkit_version:
  - 2022.09.*
  - 2022.03.*
  - 2021.09.*
```

?

Ideally we should try to minimize the number of rdkit versions and only include the last 2-3.

File: recipe/meta.yaml [hunk 2]
```diff
@@ -17,20 +17,20 @@ build:
...
-    - python                                 # [build_platform != target_platform]
+    - python {{ python_version }}            # [build_platform != target_platform]
     - cross-python_{{ target_platform }}     # [build_platform != target_platform]
     - {{ compiler('cxx') }}
     - cmake
     - make  # [unix]
     - pkg-config
   host:
-    - python
+    - python {{ python_version }}
```
Comment (hadim) [inline recipe/meta.yaml:27]: you can remove `{{ python_version }}` after tweaking the build config file.

File: recipe/conda_build_config.yaml (hunk 1 above)
Comment (hadim) [reply recipe/conda_build_config.yaml:48]: You'll need to rerender ofc after that.

Comment (hadim) [reply recipe/conda_build_config.yaml:48]: If some combinations are not desired, we will exclude them later in the meta recipe file.

**Comment by ljn917** [comment]
Replaced by #9

**Pull request closed without merging**